igvc_navigation/config/teb_local_planner_params.yaml

```diff
@@ -73,14 +79,14 @@ TebLocalPlannerROS:
   weight_inflation: 0.2
   weight_dynamic_obstacle: 10
   weight_dynamic_obstacle_inflation: 0.2
-  weight_viapoint: 1
+  weight_viapoint: 0.0001
   weight_adapt_factor: 2
 
   # Homotopy Class Planner
 
   enable_homotopy_class_planning: False
 #  enable_multithreading: True
-#  max_number_classes: 4
+#  max_number_classes: 2
```

Comment:
Not sure what the point of this was, but we'll probably enable this after #538 is resolved.

Reply:
Yeah thats true. I'll revert the change.